site/content/en/guides/consumer/function/export/tekton/_index.md

```diff
@@ -19,18 +21,19 @@ mkdir function-export-example
 cd function-export-example
 ```
 
-All commands must be run at the root of this directory.
+**All commands must be run at the root of this directory**.
 
 Use `kpt pkg get` to fetch source files of this tutorial:
 
 ```shell script
-kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/function-export-blueprint exmaple-package
 # Init git
 git init
 git remote add origin https://github.com/<USER>/<REPO>.git
```

Comment:
One idea I've seen used in other docs is to define env variables for USER and REPO at the beginning. Later commands that use $USER and $REPO can be directly copy/pasted and run correctly.

Reply:
The idea is really great. But in my case, the two env variables are only used once per tutorial.
One exception is that the generated Tekton pipeline needs the link to the repo. But it is in the file content, and not easy to read from env variables.
So I prefer to keep it simple.